install_script.sh.template

```diff
@@ -104,6 +104,39 @@ function report_telemetry() {
     safe_agent_version=$(echo -n "$agent_major_version" | tr '\n' ' ' | tr '"' '_')
   fi
 
+  library_telemetry_info=
+
+  if [ ${#apm_libraries[@]} -gt 0 ]; then
+    for lib in "${apm_libraries[@]}"
+    do
+      cur_lib=$(echo "$lib" | cut -s -d':' -f1)
+      cur_version=$(echo "$lib" | cut -s -d':' -f2)
+      if [ -z "$cur_version" ] ; then
+        cur_version="default"
+        cur_lib="$lib"
+      fi
+      case $cur_lib in
+        datadog-apm-library-java)
+          library_telemetry_info="${library_telemetry_info},\"specified_java_lib_version\": \"${cur_version}\""
+          ;;
+        datadog-apm-library-js)
+          library_telemetry_info="${library_telemetry_info},\"specified_nodejs_lib_version\": \"${cur_version}\""
+          ;;
+        datadog-apm-library-python)
+          library_telemetry_info="${library_telemetry_info},\"specified_python_lib_version\": \"${cur_version}\""
+          ;;
+        datadog-apm-library-dotnet)
+          library_telemetry_info="${library_telemetry_info},\"specified_dotnet_lib_version\": \"${cur_version}\""
+          ;;
+        datadog-apm-library-ruby)
+          library_telemetry_info="${library_telemetry_info},\"specified_ruby_lib_version\": \"${cur_version}\""
```

Comment:
it feels like this block could be simplified by matching on the last segment on the string, something like this:
```
function get_apm_lib {
   case $1 in
  datadog-apm-library-*)
    case $1 in
      *-java|*-python|*-dotnet|*-ruby)
          echo ${1#datadog-apm-library}
          ;;
      *-js)
          echo nodejs
          ;;
      *)
         fail
    esac
     ;;
  *)
   fail
   ;;
esac
}

...

library_telemetry_info="${library_telemetry_info},\"specified_$(get_apm_lib $cur_lib)_lib_version\": \"${cur_version}\""
```

The same function could be reused in the similar block below